.github/workflows/github.yml

```diff
@@ -268,6 +268,13 @@ jobs:
       run: |
         ctest --preset ${{matrix.preset}}
 
+    - name: Kill XProtect to work around CPack issue on macOS 
+      if: runner.os == 'macOS'
```

Comment:
condition should be "when building for macOS platform", as cpack for iOS doesn't use hdiutil